testing/TestHarness/TestHarness/Ext/Navigation/NavigationView/NavigationViewHomePage.xaml.cs

```diff
@@ -1,19 +1,20 @@
 ﻿
+using Uno.Extensions.Navigation.UI;
+
 namespace TestHarness.Ext.Navigation.NavigationView;
 
 public sealed partial class NavigationViewHomePage : Page
 {
 	public NavigationViewHomePage()
 	{
 		this.InitializeComponent();
+		this.Loaded += NavigationViewHomePage_Loaded;
 	}
 
-	public void NavigationItemInvoked(Microsoft.UI.Xaml.Controls.NavigationView sender, Microsoft.UI.Xaml.Controls.NavigationViewItemInvokedEventArgs e)
```

Comment:
Do we still need the NavigationItemInvoked? I think it's still reference in the XAML

Reply:
Ah, good catch! Removed it just before committing 😁